docs/examples.md

```diff
@@ -42,6 +43,18 @@ Three different protocols/workflows are illustrated:
 
 The `caprieval` module is called at various stages during the workflow to assess the quality of the models with respect to the known reference structure.
 
+## docking-nanobody-antigen
```

Comment:
A rather lengthy description compared to the other tutorials. May-be shorten it and add instead a README file into the directory with all the details.

Reply:
agreed and done!